install_config/upgrading/manual_upgrades.adoc

```diff
@@ -708,6 +708,32 @@ upgrade will fail and should be restarted automatically. This will not disrupt
 pods that are already running.
 ====
 
+[[upgading-the-registry-configuration-file]]
+=== Update your configuration file
+
+You may safely skip this part if you don't use a custom configuration file.
+
+The new registry requires following entries in the
```

Comment:
We should be specific about what "new" means here, and conditionalize appropriately for Origin vs Enterprise if needed.